solana/modules/wormhole-anchor-sdk/Cargo.toml

```diff
@@ -13,6 +13,8 @@ default = ["mainnet"]
 mainnet = []
 solana-devnet = []
 tilt-devnet = []
+fogo-devnet = []
```

Comment:
this should be `fogo-testnet` as it is referred to in their rpc and [explorer](https://explorer.fogo.io/address/78HdStBqCMioGii9D8mF3zQaWDqDZBQWTUwjjpdmbJKX?cluster=testnet)